lib/puppetlabs_spec_helper/tasks/fixtures.rb

```diff
@@ -1,409 +1,414 @@
 # frozen_string_literal: true
 
+# rubocop:disable Layout/ClassAndModuleChildren
```

Comment:
I think Rubocop does this weird thing where you also have to enable it again. I'm not sure if that's per-file or global though. Do you know? Should we disable this in the Rubocop config instead?

Reply:
I feel like it's reasonable to move this to the config. It's frustrating at best of times so to have it removed would be good.